### ISSUE TITLE
Request - Fingerslide Radius Parameter

### ISSUE THREAD

**Issue by Brausmith** [vector76/gridfinity_openscad#26]
The Fingerslide radius is considerably smaller than what is on the original, making it difficult for my sausage fingers to slide a component up the side.

Could I please request a new parameter for _Fingerslide Radius_?
Or, maybe as a fraction of the Y distance, like 1/2, 1/4, 1/8?

**Comment by wakeupgeoff** [comment]
I agree, this would be a great option. Even if it's just a drop down with Small, Medium, Large options for finger slide.

**Comment by ostat** [comment]
I added this suggestion to Gridfinity Extended (which is built on this library). You can enter the radius, and select the finger style
https://github.com/ostat/gridfinity_extended_openscad
